packages/discord-request/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "discord-request",
-	"version": "0.0.4",
```

Comment:
Instead of this, run `npm run changelog` to generate a changelog that automatically handles releases